config/src/config/execution_config.rs

```diff
@@ -136,7 +136,13 @@ mod test {
     #[test]
     fn test_some_and_load_genesis() {
         let fake_genesis = Transaction::GenesisTransaction(WriteSetPayload::Direct(
-            ChangeSet::new(WriteSetMut::new(vec![]).freeze().unwrap(), vec![], 3).unwrap(),
+            ChangeSet::new(
+                WriteSetMut::new(vec![]).freeze().unwrap(),
+                vec![],
+                // TODO(gas): probably move LATEST_GAS_FEATURE_VERSION to global-constants
```

Comment:
remove?

Reply:
? this is to address magic number 3 used below.